sdk/python/feast/infra/online_stores/contrib/hazelcast_online_store/hazelcast_online_store.py

```diff
@@ -0,0 +1,252 @@
+import logging
+from datetime import datetime
+from typing import Any, Callable, Dict, List, Literal, Optional, Sequence, Tuple
+
+import pytz
+from hazelcast.client import HazelcastClient
+from hazelcast.core import HazelcastJsonValue
+from pydantic import StrictStr
+
+from feast import Entity, FeatureView, RepoConfig
+from feast.infra.key_encoding_utils import serialize_entity_key
+from feast.infra.online_stores.online_store import OnlineStore
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
+from feast.protos.feast.types.Value_pb2 import Value as ValueProto
+from feast.repo_config import FeastConfigBaseModel
+from feast.usage import log_exceptions_and_usage
+
+
+class HazelcastOnlineStoreConfig(FeastConfigBaseModel):
+    """Online store config for Hazelcast store"""
+
+    type: Literal["hazelcast"] = "hazelcast"
+    """Online store type selector"""
+
+    cluster_name: StrictStr = "dev"
+    """Name of the cluster you want to connect. The default cluster name is `dev`"""
+
+    cluster_members: Optional[List] = ["localhost:5701"]
```

Comment:
The type should be `Optional[List[str]]` ?

Reply:
Fixed at https://github.com/mehmettokgoz/feast/pull/1/commits/9f8174c72faf2bbdbd6f7fe4b414d63f20338600#diff-47764bf85c9631f63e542e314d1d29579a85e570b2813040caaade67743322d7R64